### DISCUSSION TITLE
[@types/three] missing TransformControlsGizmo

### DISCUSSION THREAD

**Discussion by mike-wood-dywidag** [DefinitelyTyped/DefinitelyTyped#59536] [Issues with a @types package]
Hi,
The THREE js library is missing TransformControlsGizmo and TransformControlsPlane from [TransformControls.js](https://github.com/mrdoob/three.js/blob/4e97ffa91ae782b5fd4b12735f301740953f5abd/examples/jsm/controls/TransformControls.js) exports

Many thanks

[a comment by an automated account is not shown]

**Comment by joshuaellis** [comment]
Please open this issue // create a PR in [three-ts-types](https://github.com/three-types/three-ts-types)
